### DISCUSSION TITLE
Choose a MF Variable directly as a Placeholder/Precondition

### DISCUSSION THREAD

**Discussion by pizman** [MobiFlight/MobiFlight-Connector#493] [Ideas]
[https://github.com/MobiFlight/MobiFlight-Connector/discussions/382#discussioncomment-932953](url)

I still wrote this idea at beginning of the "MF own Variables" thematic.

In the past we had to READ a Varibale Data ( Custom Offsets from FSUIPC)  via a Output Config.....  And then we was able to use THIS Output Config Item  inside another Config as a Placeholder or a Precondition.

NOW with our internal Memory this "bypass" is not logical i think.

Lets say i like to nuse the MF Varibale "Test1" as a Precondition or Placeholder.....
Then i still need a "Fake Output Config" that just Read this MF Variable.... So i can use that Config Value later for other things.
So Output Config "Alpha" Reads "Test1" Variable.....    And then we can use "Alpha" as a Placeholder or Precon.

Solution:
In Config Refferences AND Precondition we should be able to choose from all exisiting Output Configs  ( like now) AND additional all exisitin MF Variables ..... So in that example we could choose "Test1" directly there without read it before.

**Comment by MobiFlight-Admin** [comment]
Great idea! I will add this when I will add the button state.

**Comment by DocMoebiuz** [comment]
See https://github.com/MobiFlight/MobiFlight-Connector/issues/545
